functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -99,10 +150,13 @@ Then it will export the results to Export-DbaDiagnosticQuery.
         [Switch]$NoPlanColumn,
         [Switch]$NoColumnParsing,
         [switch][Alias('Silent')]
```

Comment:
last param should always be 
```
        [Alias('Silent')]
        [switch]$EnableException
```
for functions coming before the great-silent-rewrite. New functions can just use
```
[switch]$EnableException
```